lib/gpx.js

```diff
@@ -0,0 +1,79 @@
+'use strict'
+
+var gpxParse = require("gpx-parse");
+var attributes = require('./attributes'),
+    sportAttr = attributes.sport,
+    trackAttr = attributes.track
+
+var EARTH_RADIUS_KM = 6371.0;
```

Comment:
This file contains quite a bit of logic, that i am sure its already done by some package (i think i have used in a different project. If you want we can talk about it or i can look at it to refactor on the future